fluent-syntax/src/parser.js

```diff
@@ -269,6 +269,7 @@ export default class FluentParser {
     // XXX Replace with ps.expectChar('=').
     if (ps.currentIs("=")) {
       ps.next();
+      ps.skipInlineWS();
```

Comment:
I'm not sure if I understand this change. You're skipping inline WS, but then `isPeekPatternStart` also peeks through it and then `skipIndent` skips it.

So, in which scenario it is needed?

Reply:
We need to `skipInlineWS` regardless of whether there's a Pattern or not. Would this be more readable if I moved `skipInlineWS` into an `else` block of the `isPeekPatternStart` check?